File: backend/Common/ViewModels/FeedbackVMs/FeedbackVM.cs
namespace Common.ViewModels.FeedbackVMs;
public class FeedbackVM
{
    public int Id { get; set; }
    public string Avatar { get; set; }
    public string Name { get; set; }
    public string CreatedDate { get; set; }
    public int Rating { get; set; }
    public string Content { get; set; }
    public string Response { get; set; }
    public int SenderId { get; set; }
    public int OwnerId { get; set; }
}

File: frontend/src/models/viewmodels/feedback.vm.ts
import { PaginationData } from "../app.interface";

export interface FeedbackVM {
  id: number;
  avatar: string;
  name: string;
  createdDate: string;
  rating: number;
  content: string;
  response?: string;
  senderId: number;
  ownerId: number;
}

export interface FeedbackInfoVM {
  feedbacks: FeedbackVM[];
  averageRating: number;
  totalFeedback: number;
}

export interface PagedFeedbacksVM {
  feedbacks: FeedbackVM[];
  averageRating: number;
  totalFeedback: number;
  pagination: PaginationData;
}
